CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ changes.
 
 - Tested with `cardano-node 9.2.0` and `cardano-cli 9.4.1.0`.
 
+- **BREAKING** Rewrite of the commit script in aiken:
+  - This makes `abort` and `collectCom` transactions more efficient and results
+    in a new maximum number of head participants being `8`.
```

Comment:
Is it 8 or 9? The pr description says 9 but here we have 8.

Reply:
It was 9 first, but needed to change it to 8 in https://github.com/cardano-scaling/hydra/pull/1680/commits/665cc1789daa10065f3ae5437fcfeba9100860ee. I'll update the PR description too.